lib/flowViewport/sceneIndex/fvpIsolateSelectSceneIndex.cpp

```diff
@@ -454,6 +487,33 @@ IsolateSelectSceneIndex::_CollectInstancers(
     return instancers;
 }
 
+bool IsolateSelectSceneIndex::_IsPointInstancer(
+    const PXR_NS::SdfPath& primPath
+) const
+{
+    auto prim = GetInputSceneIndex()->GetPrim(primPath);
```

Comment:
Is there a reason for not using the FVP prim utils IsPointInstancer function ?

Reply:
No, that was the whole point of factoring it out, done.